inst/shinyGUI/ui.R

```diff
@@ -26,7 +26,7 @@ medIntensities <- fluidPage(
       ## Plot 1
       h2("Multi-dimensional scaling plot computed using median protein expression"),
       ## Need to incl library("shinycssloaders") into the package
-      withSpinner(plotOutput(outputId="plotMDS", width="100%", height = "600px"), type = 2),
+      withSpinner(plotOutput(outputId="plotMDS", height = "600px"), type = 2),
```

Comment:
Removed Width as the inclusion of width causes size hardcoding, making it less dynamic